ERROR - 2024-09-06 05:49:49 --> mysqli_sql_exception: Duplicate entry '1' for key 'editorial_id' in C:\xampp\htdocs\parcial1_pp\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp\htdocs\parcial1_pp\system\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `li...', 0)
#1 C:\xampp\htdocs\parcial1_pp\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `li...')
#2 C:\xampp\htdocs\parcial1_pp\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `li...')
#3 C:\xampp\htdocs\parcial1_pp\system\Database\BaseBuilder.php(2332): CodeIgniter\Database\BaseConnection->query('INSERT INTO `li...', Array, false)
#4 C:\xampp\htdocs\parcial1_pp\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\parcial1_pp\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\parcial1_pp\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\parcial1_pp\app\Controllers\Libros.php(45): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\parcial1_pp\system\CodeIgniter.php(933): App\Controllers\Libros->insertar()
#9 C:\xampp\htdocs\parcial1_pp\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Libros))
#10 C:\xampp\htdocs\parcial1_pp\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\parcial1_pp\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\parcial1_pp\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\parcial1_pp\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2024-09-06 05:49:49 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '1' for key 'editorial_id'
[Method: POST, Route: insertar]
in SYSTEMPATH\Database\BaseConnection.php on line 688.
 1 SYSTEMPATH\Database\BaseBuilder.php(2332): CodeIgniter\Database\BaseConnection->query('INSERT INTO `libros` (`titulo`, `fecha_publicacion`, `precio`, `editorial_id`, `isbn`) VALUES (:titulo:, :fecha_publicacion:, :precio:, :editorial_id:, :isbn:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Libros.php(45): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Libros->insertar()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Libros))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-09-06 05:49:49 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '1' for key 'editorial_id'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 317.
 1 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `libros` (`titulo`, `fecha_publicacion`, `precio`, `editorial_id`, `isbn`) VALUES (\'Lo que el viento se llevo2\', \'1996-06-06\', \'1236\', \'1\', \'565asda\')')
 2 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `libros` (`titulo`, `fecha_publicacion`, `precio`, `editorial_id`, `isbn`) VALUES (\'Lo que el viento se llevo2\', \'1996-06-06\', \'1236\', \'1\', \'565asda\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2332): CodeIgniter\Database\BaseConnection->query('INSERT INTO `libros` (`titulo`, `fecha_publicacion`, `precio`, `editorial_id`, `isbn`) VALUES (:titulo:, :fecha_publicacion:, :precio:, :editorial_id:, :isbn:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\Libros.php(45): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Libros->insertar()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Libros))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-09-06 05:49:49 --> [Caused by] mysqli_sql_exception: Duplicate entry '1' for key 'editorial_id'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 312.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `libros` (`titulo`, `fecha_publicacion`, `precio`, `editorial_id`, `isbn`) VALUES (\'Lo que el viento se llevo2\', \'1996-06-06\', \'1236\', \'1\', \'565asda\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `libros` (`titulo`, `fecha_publicacion`, `precio`, `editorial_id`, `isbn`) VALUES (\'Lo que el viento se llevo2\', \'1996-06-06\', \'1236\', \'1\', \'565asda\')')
 3 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `libros` (`titulo`, `fecha_publicacion`, `precio`, `editorial_id`, `isbn`) VALUES (\'Lo que el viento se llevo2\', \'1996-06-06\', \'1236\', \'1\', \'565asda\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2332): CodeIgniter\Database\BaseConnection->query('INSERT INTO `libros` (`titulo`, `fecha_publicacion`, `precio`, `editorial_id`, `isbn`) VALUES (:titulo:, :fecha_publicacion:, :precio:, :editorial_id:, :isbn:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Libros.php(45): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Libros->insertar()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Libros))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-09-06 05:50:34 --> mysqli_sql_exception: Duplicate entry '1' for key 'editorial_id' in C:\xampp\htdocs\parcial1_pp\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp\htdocs\parcial1_pp\system\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `li...', 0)
#1 C:\xampp\htdocs\parcial1_pp\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `li...')
#2 C:\xampp\htdocs\parcial1_pp\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `li...')
#3 C:\xampp\htdocs\parcial1_pp\system\Database\BaseBuilder.php(2332): CodeIgniter\Database\BaseConnection->query('INSERT INTO `li...', Array, false)
#4 C:\xampp\htdocs\parcial1_pp\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\parcial1_pp\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\parcial1_pp\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\parcial1_pp\app\Controllers\Libros.php(45): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\parcial1_pp\system\CodeIgniter.php(933): App\Controllers\Libros->insertar()
#9 C:\xampp\htdocs\parcial1_pp\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Libros))
#10 C:\xampp\htdocs\parcial1_pp\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\parcial1_pp\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\parcial1_pp\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\parcial1_pp\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2024-09-06 05:50:34 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '1' for key 'editorial_id'
[Method: POST, Route: insertar]
in SYSTEMPATH\Database\BaseConnection.php on line 688.
 1 SYSTEMPATH\Database\BaseBuilder.php(2332): CodeIgniter\Database\BaseConnection->query('INSERT INTO `libros` (`titulo`, `fecha_publicacion`, `precio`, `editorial_id`, `isbn`) VALUES (:titulo:, :fecha_publicacion:, :precio:, :editorial_id:, :isbn:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Libros.php(45): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Libros->insertar()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Libros))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-09-06 05:50:34 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '1' for key 'editorial_id'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 317.
 1 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `libros` (`titulo`, `fecha_publicacion`, `precio`, `editorial_id`, `isbn`) VALUES (\'Lo que el viento se llevo2\', \'1996-06-06\', \'1236\', \'1\', \'565asda\')')
 2 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `libros` (`titulo`, `fecha_publicacion`, `precio`, `editorial_id`, `isbn`) VALUES (\'Lo que el viento se llevo2\', \'1996-06-06\', \'1236\', \'1\', \'565asda\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2332): CodeIgniter\Database\BaseConnection->query('INSERT INTO `libros` (`titulo`, `fecha_publicacion`, `precio`, `editorial_id`, `isbn`) VALUES (:titulo:, :fecha_publicacion:, :precio:, :editorial_id:, :isbn:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\Libros.php(45): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Libros->insertar()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Libros))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-09-06 05:50:34 --> [Caused by] mysqli_sql_exception: Duplicate entry '1' for key 'editorial_id'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 312.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `libros` (`titulo`, `fecha_publicacion`, `precio`, `editorial_id`, `isbn`) VALUES (\'Lo que el viento se llevo2\', \'1996-06-06\', \'1236\', \'1\', \'565asda\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `libros` (`titulo`, `fecha_publicacion`, `precio`, `editorial_id`, `isbn`) VALUES (\'Lo que el viento se llevo2\', \'1996-06-06\', \'1236\', \'1\', \'565asda\')')
 3 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `libros` (`titulo`, `fecha_publicacion`, `precio`, `editorial_id`, `isbn`) VALUES (\'Lo que el viento se llevo2\', \'1996-06-06\', \'1236\', \'1\', \'565asda\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2332): CodeIgniter\Database\BaseConnection->query('INSERT INTO `libros` (`titulo`, `fecha_publicacion`, `precio`, `editorial_id`, `isbn`) VALUES (:titulo:, :fecha_publicacion:, :precio:, :editorial_id:, :isbn:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Libros.php(45): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Libros->insertar()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Libros))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-09-06 05:52:07 --> mysqli_sql_exception: Duplicate entry '1' for key 'editorial_id' in C:\xampp\htdocs\parcial1_pp\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp\htdocs\parcial1_pp\system\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `li...', 0)
#1 C:\xampp\htdocs\parcial1_pp\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `li...')
#2 C:\xampp\htdocs\parcial1_pp\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `li...')
#3 C:\xampp\htdocs\parcial1_pp\system\Database\BaseBuilder.php(2332): CodeIgniter\Database\BaseConnection->query('INSERT INTO `li...', Array, false)
#4 C:\xampp\htdocs\parcial1_pp\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\parcial1_pp\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\parcial1_pp\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\parcial1_pp\app\Controllers\Libros.php(45): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\parcial1_pp\system\CodeIgniter.php(933): App\Controllers\Libros->insertar()
#9 C:\xampp\htdocs\parcial1_pp\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Libros))
#10 C:\xampp\htdocs\parcial1_pp\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\parcial1_pp\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\parcial1_pp\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\parcial1_pp\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2024-09-06 05:52:07 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '1' for key 'editorial_id'
[Method: POST, Route: insertar]
in SYSTEMPATH\Database\BaseConnection.php on line 688.
 1 SYSTEMPATH\Database\BaseBuilder.php(2332): CodeIgniter\Database\BaseConnection->query('INSERT INTO `libros` (`titulo`, `fecha_publicacion`, `precio`, `editorial_id`, `isbn`) VALUES (:titulo:, :fecha_publicacion:, :precio:, :editorial_id:, :isbn:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Libros.php(45): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Libros->insertar()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Libros))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-09-06 05:52:07 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '1' for key 'editorial_id'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 317.
 1 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `libros` (`titulo`, `fecha_publicacion`, `precio`, `editorial_id`, `isbn`) VALUES (\'Lo que el viento se llevo 8\', \'1990-06-09\', \'12\', \'1\', \'ASDF\')')
 2 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `libros` (`titulo`, `fecha_publicacion`, `precio`, `editorial_id`, `isbn`) VALUES (\'Lo que el viento se llevo 8\', \'1990-06-09\', \'12\', \'1\', \'ASDF\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2332): CodeIgniter\Database\BaseConnection->query('INSERT INTO `libros` (`titulo`, `fecha_publicacion`, `precio`, `editorial_id`, `isbn`) VALUES (:titulo:, :fecha_publicacion:, :precio:, :editorial_id:, :isbn:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\Libros.php(45): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Libros->insertar()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Libros))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-09-06 05:52:07 --> [Caused by] mysqli_sql_exception: Duplicate entry '1' for key 'editorial_id'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 312.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `libros` (`titulo`, `fecha_publicacion`, `precio`, `editorial_id`, `isbn`) VALUES (\'Lo que el viento se llevo 8\', \'1990-06-09\', \'12\', \'1\', \'ASDF\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `libros` (`titulo`, `fecha_publicacion`, `precio`, `editorial_id`, `isbn`) VALUES (\'Lo que el viento se llevo 8\', \'1990-06-09\', \'12\', \'1\', \'ASDF\')')
 3 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `libros` (`titulo`, `fecha_publicacion`, `precio`, `editorial_id`, `isbn`) VALUES (\'Lo que el viento se llevo 8\', \'1990-06-09\', \'12\', \'1\', \'ASDF\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2332): CodeIgniter\Database\BaseConnection->query('INSERT INTO `libros` (`titulo`, `fecha_publicacion`, `precio`, `editorial_id`, `isbn`) VALUES (:titulo:, :fecha_publicacion:, :precio:, :editorial_id:, :isbn:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Libros.php(45): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Libros->insertar()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Libros))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-09-06 05:52:36 --> mysqli_sql_exception: Duplicate entry '1' for key 'editorial_id' in C:\xampp\htdocs\parcial1_pp\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp\htdocs\parcial1_pp\system\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `li...', 0)
#1 C:\xampp\htdocs\parcial1_pp\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `li...')
#2 C:\xampp\htdocs\parcial1_pp\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `li...')
#3 C:\xampp\htdocs\parcial1_pp\system\Database\BaseBuilder.php(2332): CodeIgniter\Database\BaseConnection->query('INSERT INTO `li...', Array, false)
#4 C:\xampp\htdocs\parcial1_pp\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\parcial1_pp\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\parcial1_pp\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\parcial1_pp\app\Controllers\Libros.php(45): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\parcial1_pp\system\CodeIgniter.php(933): App\Controllers\Libros->insertar()
#9 C:\xampp\htdocs\parcial1_pp\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Libros))
#10 C:\xampp\htdocs\parcial1_pp\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\parcial1_pp\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\parcial1_pp\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\parcial1_pp\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2024-09-06 05:52:36 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '1' for key 'editorial_id'
[Method: POST, Route: insertar]
in SYSTEMPATH\Database\BaseConnection.php on line 688.
 1 SYSTEMPATH\Database\BaseBuilder.php(2332): CodeIgniter\Database\BaseConnection->query('INSERT INTO `libros` (`titulo`, `fecha_publicacion`, `precio`, `editorial_id`, `isbn`) VALUES (:titulo:, :fecha_publicacion:, :precio:, :editorial_id:, :isbn:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Libros.php(45): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Libros->insertar()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Libros))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-09-06 05:52:36 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '1' for key 'editorial_id'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 317.
 1 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `libros` (`titulo`, `fecha_publicacion`, `precio`, `editorial_id`, `isbn`) VALUES (\'Lo que el viento se llevo 8\', \'1990-06-09\', \'12\', \'1\', \'ASDF\')')
 2 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `libros` (`titulo`, `fecha_publicacion`, `precio`, `editorial_id`, `isbn`) VALUES (\'Lo que el viento se llevo 8\', \'1990-06-09\', \'12\', \'1\', \'ASDF\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2332): CodeIgniter\Database\BaseConnection->query('INSERT INTO `libros` (`titulo`, `fecha_publicacion`, `precio`, `editorial_id`, `isbn`) VALUES (:titulo:, :fecha_publicacion:, :precio:, :editorial_id:, :isbn:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\Libros.php(45): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Libros->insertar()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Libros))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-09-06 05:52:36 --> [Caused by] mysqli_sql_exception: Duplicate entry '1' for key 'editorial_id'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 312.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `libros` (`titulo`, `fecha_publicacion`, `precio`, `editorial_id`, `isbn`) VALUES (\'Lo que el viento se llevo 8\', \'1990-06-09\', \'12\', \'1\', \'ASDF\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `libros` (`titulo`, `fecha_publicacion`, `precio`, `editorial_id`, `isbn`) VALUES (\'Lo que el viento se llevo 8\', \'1990-06-09\', \'12\', \'1\', \'ASDF\')')
 3 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `libros` (`titulo`, `fecha_publicacion`, `precio`, `editorial_id`, `isbn`) VALUES (\'Lo que el viento se llevo 8\', \'1990-06-09\', \'12\', \'1\', \'ASDF\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2332): CodeIgniter\Database\BaseConnection->query('INSERT INTO `libros` (`titulo`, `fecha_publicacion`, `precio`, `editorial_id`, `isbn`) VALUES (:titulo:, :fecha_publicacion:, :precio:, :editorial_id:, :isbn:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Libros.php(45): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Libros->insertar()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Libros))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-09-06 05:53:58 --> mysqli_sql_exception: Duplicate entry '1' for key 'editorial_id' in C:\xampp\htdocs\parcial1_pp\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp\htdocs\parcial1_pp\system\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `li...', 0)
#1 C:\xampp\htdocs\parcial1_pp\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `li...')
#2 C:\xampp\htdocs\parcial1_pp\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `li...')
#3 C:\xampp\htdocs\parcial1_pp\system\Database\BaseBuilder.php(2332): CodeIgniter\Database\BaseConnection->query('INSERT INTO `li...', Array, false)
#4 C:\xampp\htdocs\parcial1_pp\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\parcial1_pp\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\parcial1_pp\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\parcial1_pp\app\Controllers\Libros.php(45): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\parcial1_pp\system\CodeIgniter.php(933): App\Controllers\Libros->insertar()
#9 C:\xampp\htdocs\parcial1_pp\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Libros))
#10 C:\xampp\htdocs\parcial1_pp\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\parcial1_pp\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\parcial1_pp\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\parcial1_pp\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2024-09-06 05:53:58 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '1' for key 'editorial_id'
[Method: POST, Route: insertar]
in SYSTEMPATH\Database\BaseConnection.php on line 688.
 1 SYSTEMPATH\Database\BaseBuilder.php(2332): CodeIgniter\Database\BaseConnection->query('INSERT INTO `libros` (`titulo`, `fecha_publicacion`, `precio`, `editorial_id`, `isbn`) VALUES (:titulo:, :fecha_publicacion:, :precio:, :editorial_id:, :isbn:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Libros.php(45): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Libros->insertar()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Libros))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-09-06 05:53:58 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '1' for key 'editorial_id'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 317.
 1 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `libros` (`titulo`, `fecha_publicacion`, `precio`, `editorial_id`, `isbn`) VALUES (\'Lo que el viento se llevo 8\', \'1990-06-09\', \'12\', \'1\', \'ASDF\')')
 2 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `libros` (`titulo`, `fecha_publicacion`, `precio`, `editorial_id`, `isbn`) VALUES (\'Lo que el viento se llevo 8\', \'1990-06-09\', \'12\', \'1\', \'ASDF\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2332): CodeIgniter\Database\BaseConnection->query('INSERT INTO `libros` (`titulo`, `fecha_publicacion`, `precio`, `editorial_id`, `isbn`) VALUES (:titulo:, :fecha_publicacion:, :precio:, :editorial_id:, :isbn:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\Libros.php(45): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Libros->insertar()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Libros))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-09-06 05:53:58 --> [Caused by] mysqli_sql_exception: Duplicate entry '1' for key 'editorial_id'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 312.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `libros` (`titulo`, `fecha_publicacion`, `precio`, `editorial_id`, `isbn`) VALUES (\'Lo que el viento se llevo 8\', \'1990-06-09\', \'12\', \'1\', \'ASDF\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `libros` (`titulo`, `fecha_publicacion`, `precio`, `editorial_id`, `isbn`) VALUES (\'Lo que el viento se llevo 8\', \'1990-06-09\', \'12\', \'1\', \'ASDF\')')
 3 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `libros` (`titulo`, `fecha_publicacion`, `precio`, `editorial_id`, `isbn`) VALUES (\'Lo que el viento se llevo 8\', \'1990-06-09\', \'12\', \'1\', \'ASDF\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2332): CodeIgniter\Database\BaseConnection->query('INSERT INTO `libros` (`titulo`, `fecha_publicacion`, `precio`, `editorial_id`, `isbn`) VALUES (:titulo:, :fecha_publicacion:, :precio:, :editorial_id:, :isbn:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Libros.php(45): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Libros->insertar()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Libros))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-09-06 05:55:53 --> mysqli_sql_exception: Duplicate entry '1' for key 'editorial_id' in C:\xampp\htdocs\parcial1_pp\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp\htdocs\parcial1_pp\system\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `li...', 0)
#1 C:\xampp\htdocs\parcial1_pp\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `li...')
#2 C:\xampp\htdocs\parcial1_pp\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `li...')
#3 C:\xampp\htdocs\parcial1_pp\system\Database\BaseBuilder.php(2332): CodeIgniter\Database\BaseConnection->query('INSERT INTO `li...', Array, false)
#4 C:\xampp\htdocs\parcial1_pp\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\parcial1_pp\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\parcial1_pp\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\parcial1_pp\app\Controllers\Libros.php(45): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\parcial1_pp\system\CodeIgniter.php(933): App\Controllers\Libros->insertar()
#9 C:\xampp\htdocs\parcial1_pp\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Libros))
#10 C:\xampp\htdocs\parcial1_pp\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\parcial1_pp\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\parcial1_pp\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\parcial1_pp\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2024-09-06 05:55:53 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '1' for key 'editorial_id'
[Method: POST, Route: insertar]
in SYSTEMPATH\Database\BaseConnection.php on line 688.
 1 SYSTEMPATH\Database\BaseBuilder.php(2332): CodeIgniter\Database\BaseConnection->query('INSERT INTO `libros` (`titulo`, `fecha_publicacion`, `precio`, `editorial_id`, `isbn`) VALUES (:titulo:, :fecha_publicacion:, :precio:, :editorial_id:, :isbn:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Libros.php(45): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Libros->insertar()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Libros))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-09-06 05:55:53 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '1' for key 'editorial_id'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 317.
 1 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `libros` (`titulo`, `fecha_publicacion`, `precio`, `editorial_id`, `isbn`) VALUES (\'Lo que el viento se llevó 9\', \'1944-06-19\', \'44\', \'1\', \'ABCS643\')')
 2 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `libros` (`titulo`, `fecha_publicacion`, `precio`, `editorial_id`, `isbn`) VALUES (\'Lo que el viento se llevó 9\', \'1944-06-19\', \'44\', \'1\', \'ABCS643\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2332): CodeIgniter\Database\BaseConnection->query('INSERT INTO `libros` (`titulo`, `fecha_publicacion`, `precio`, `editorial_id`, `isbn`) VALUES (:titulo:, :fecha_publicacion:, :precio:, :editorial_id:, :isbn:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\Libros.php(45): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Libros->insertar()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Libros))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-09-06 05:55:53 --> [Caused by] mysqli_sql_exception: Duplicate entry '1' for key 'editorial_id'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 312.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `libros` (`titulo`, `fecha_publicacion`, `precio`, `editorial_id`, `isbn`) VALUES (\'Lo que el viento se llevó 9\', \'1944-06-19\', \'44\', \'1\', \'ABCS643\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `libros` (`titulo`, `fecha_publicacion`, `precio`, `editorial_id`, `isbn`) VALUES (\'Lo que el viento se llevó 9\', \'1944-06-19\', \'44\', \'1\', \'ABCS643\')')
 3 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `libros` (`titulo`, `fecha_publicacion`, `precio`, `editorial_id`, `isbn`) VALUES (\'Lo que el viento se llevó 9\', \'1944-06-19\', \'44\', \'1\', \'ABCS643\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2332): CodeIgniter\Database\BaseConnection->query('INSERT INTO `libros` (`titulo`, `fecha_publicacion`, `precio`, `editorial_id`, `isbn`) VALUES (:titulo:, :fecha_publicacion:, :precio:, :editorial_id:, :isbn:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Libros.php(45): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Libros->insertar()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Libros))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-09-06 05:56:00 --> mysqli_sql_exception: Duplicate entry '4' for key 'editorial_id' in C:\xampp\htdocs\parcial1_pp\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp\htdocs\parcial1_pp\system\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `li...', 0)
#1 C:\xampp\htdocs\parcial1_pp\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `li...')
#2 C:\xampp\htdocs\parcial1_pp\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `li...')
#3 C:\xampp\htdocs\parcial1_pp\system\Database\BaseBuilder.php(2332): CodeIgniter\Database\BaseConnection->query('INSERT INTO `li...', Array, false)
#4 C:\xampp\htdocs\parcial1_pp\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\parcial1_pp\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\parcial1_pp\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\parcial1_pp\app\Controllers\Libros.php(45): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\parcial1_pp\system\CodeIgniter.php(933): App\Controllers\Libros->insertar()
#9 C:\xampp\htdocs\parcial1_pp\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Libros))
#10 C:\xampp\htdocs\parcial1_pp\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\parcial1_pp\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\parcial1_pp\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\parcial1_pp\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2024-09-06 05:56:00 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '4' for key 'editorial_id'
[Method: POST, Route: insertar]
in SYSTEMPATH\Database\BaseConnection.php on line 688.
 1 SYSTEMPATH\Database\BaseBuilder.php(2332): CodeIgniter\Database\BaseConnection->query('INSERT INTO `libros` (`titulo`, `fecha_publicacion`, `precio`, `editorial_id`, `isbn`) VALUES (:titulo:, :fecha_publicacion:, :precio:, :editorial_id:, :isbn:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Libros.php(45): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Libros->insertar()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Libros))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-09-06 05:56:00 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '4' for key 'editorial_id'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 317.
 1 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `libros` (`titulo`, `fecha_publicacion`, `precio`, `editorial_id`, `isbn`) VALUES (\'Lo que el viento se llevó 9\', \'1944-06-19\', \'44\', \'4\', \'ABCS643\')')
 2 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `libros` (`titulo`, `fecha_publicacion`, `precio`, `editorial_id`, `isbn`) VALUES (\'Lo que el viento se llevó 9\', \'1944-06-19\', \'44\', \'4\', \'ABCS643\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2332): CodeIgniter\Database\BaseConnection->query('INSERT INTO `libros` (`titulo`, `fecha_publicacion`, `precio`, `editorial_id`, `isbn`) VALUES (:titulo:, :fecha_publicacion:, :precio:, :editorial_id:, :isbn:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\Libros.php(45): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Libros->insertar()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Libros))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-09-06 05:56:00 --> [Caused by] mysqli_sql_exception: Duplicate entry '4' for key 'editorial_id'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 312.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `libros` (`titulo`, `fecha_publicacion`, `precio`, `editorial_id`, `isbn`) VALUES (\'Lo que el viento se llevó 9\', \'1944-06-19\', \'44\', \'4\', \'ABCS643\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `libros` (`titulo`, `fecha_publicacion`, `precio`, `editorial_id`, `isbn`) VALUES (\'Lo que el viento se llevó 9\', \'1944-06-19\', \'44\', \'4\', \'ABCS643\')')
 3 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `libros` (`titulo`, `fecha_publicacion`, `precio`, `editorial_id`, `isbn`) VALUES (\'Lo que el viento se llevó 9\', \'1944-06-19\', \'44\', \'4\', \'ABCS643\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2332): CodeIgniter\Database\BaseConnection->query('INSERT INTO `libros` (`titulo`, `fecha_publicacion`, `precio`, `editorial_id`, `isbn`) VALUES (:titulo:, :fecha_publicacion:, :precio:, :editorial_id:, :isbn:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Libros.php(45): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Libros->insertar()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Libros))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-09-06 05:56:09 --> mysqli_sql_exception: Duplicate entry '6' for key 'editorial_id' in C:\xampp\htdocs\parcial1_pp\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp\htdocs\parcial1_pp\system\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `li...', 0)
#1 C:\xampp\htdocs\parcial1_pp\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `li...')
#2 C:\xampp\htdocs\parcial1_pp\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `li...')
#3 C:\xampp\htdocs\parcial1_pp\system\Database\BaseBuilder.php(2332): CodeIgniter\Database\BaseConnection->query('INSERT INTO `li...', Array, false)
#4 C:\xampp\htdocs\parcial1_pp\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\parcial1_pp\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\parcial1_pp\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\parcial1_pp\app\Controllers\Libros.php(45): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\parcial1_pp\system\CodeIgniter.php(933): App\Controllers\Libros->insertar()
#9 C:\xampp\htdocs\parcial1_pp\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Libros))
#10 C:\xampp\htdocs\parcial1_pp\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\parcial1_pp\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\parcial1_pp\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\parcial1_pp\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2024-09-06 05:56:09 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '6' for key 'editorial_id'
[Method: POST, Route: insertar]
in SYSTEMPATH\Database\BaseConnection.php on line 688.
 1 SYSTEMPATH\Database\BaseBuilder.php(2332): CodeIgniter\Database\BaseConnection->query('INSERT INTO `libros` (`titulo`, `fecha_publicacion`, `precio`, `editorial_id`, `isbn`) VALUES (:titulo:, :fecha_publicacion:, :precio:, :editorial_id:, :isbn:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Libros.php(45): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Libros->insertar()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Libros))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-09-06 05:56:09 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '6' for key 'editorial_id'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 317.
 1 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `libros` (`titulo`, `fecha_publicacion`, `precio`, `editorial_id`, `isbn`) VALUES (\'Lo que el viento se llevó 9\', \'1944-06-19\', \'44\', \'6\', \'ABCS643\')')
 2 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `libros` (`titulo`, `fecha_publicacion`, `precio`, `editorial_id`, `isbn`) VALUES (\'Lo que el viento se llevó 9\', \'1944-06-19\', \'44\', \'6\', \'ABCS643\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2332): CodeIgniter\Database\BaseConnection->query('INSERT INTO `libros` (`titulo`, `fecha_publicacion`, `precio`, `editorial_id`, `isbn`) VALUES (:titulo:, :fecha_publicacion:, :precio:, :editorial_id:, :isbn:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\Libros.php(45): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Libros->insertar()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Libros))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-09-06 05:56:09 --> [Caused by] mysqli_sql_exception: Duplicate entry '6' for key 'editorial_id'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 312.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `libros` (`titulo`, `fecha_publicacion`, `precio`, `editorial_id`, `isbn`) VALUES (\'Lo que el viento se llevó 9\', \'1944-06-19\', \'44\', \'6\', \'ABCS643\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `libros` (`titulo`, `fecha_publicacion`, `precio`, `editorial_id`, `isbn`) VALUES (\'Lo que el viento se llevó 9\', \'1944-06-19\', \'44\', \'6\', \'ABCS643\')')
 3 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `libros` (`titulo`, `fecha_publicacion`, `precio`, `editorial_id`, `isbn`) VALUES (\'Lo que el viento se llevó 9\', \'1944-06-19\', \'44\', \'6\', \'ABCS643\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2332): CodeIgniter\Database\BaseConnection->query('INSERT INTO `libros` (`titulo`, `fecha_publicacion`, `precio`, `editorial_id`, `isbn`) VALUES (:titulo:, :fecha_publicacion:, :precio:, :editorial_id:, :isbn:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Libros.php(45): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Libros->insertar()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Libros))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
